<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomaticGeometricAttributesDialogBase</class>
 <widget class="QDialog" name="AutomaticGeometricAttributesDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>saniHUB RedBasica</string>
  </property>
  <layout class="QFormLayout" name="formLayout_6">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbExistingLayer">
        <property name="text">
         <string>Existing Layer</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbNewLayer">
        <property name="text">
         <string>New Layer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="grpNewLayer">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Name the layer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLineEdit" name="txtLayerName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="grpExistingLayer">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select a layer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="cboLayers"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Patch</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Extension</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="cboExtension"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Begin coordinate E</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>End coordinate E</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QComboBox" name="cboBeginCoordE"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Begin coordinate N</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>End coordinate N</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QComboBox" name="cboBeginCoordN"/>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="cboEndCoordN"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Segment Name</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Numered Segment Name</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QComboBox" name="cboSegName"/>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="cboSegNameC"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Aux Pav 1</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Aux Pav 2</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QComboBox" name="cboAuxPav1"/>
          </item>
          <item row="10" column="1">
           <widget class="QComboBox" name="cboAuxPav2"/>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Aux Prof I</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Aux Prof F</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QComboBox" name="cboAuxProf1"/>
          </item>
          <item row="12" column="1">
           <widget class="QComboBox" name="cboAuxProf2"/>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label_27">
            <property name="text">
             <string>Aux 01</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Aux 02</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QComboBox" name="cboAux1"/>
          </item>
          <item row="14" column="1">
           <widget class="QComboBox" name="cboAux2"/>
          </item>
          <item row="15" column="0">
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Aux 03</string>
            </property>
           </widget>
          </item>
          <item row="15" column="1">
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>Aux Pos</string>
            </property>
           </widget>
          </item>
          <item row="16" column="0">
           <widget class="QComboBox" name="cboAux3"/>
          </item>
          <item row="16" column="1">
           <widget class="QComboBox" name="cboAuxPos"/>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cboEndCoordE"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Nodes</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Name the layer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLineEdit" name="txtLayerNodeName"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>QE</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="cboCota">
            <property name="text">
             <string>Cota</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="txtQE"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="txtCota"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
